### DISCUSSION TITLE
Moving database from Home Assistant

### DISCUSSION THREAD

**Discussion by shawwellpete** [teslamate-org/teslamate#2703] [Q&A]
Hi, firstly, thank you for the fantastic work on Tesla Mate.

I have Teslamate set up and working well on my Debian NUC which also runs home assistant. I am pretty new to docker but got it working using Portainer which I prefer as it is more visual.

I noticed that  Teslamate can  also be set up as an Add-on in home assistant which appeals to me as it would be backed up with my other Home Assistant data and I would be able to get remote access via Nabu Casa. 

My plan was to install following the instructions [here ](https://github.com/matt-FFFFFF/hassio-addon-repository/blob/main/teslamate/README.md) and then find a way of moving the database from one install to the other.

The part I fell down on was "Configure a postgresql database as follows..."

![grafana-postgres](https://user-images.githubusercontent.com/74064242/173527938-b35b6dd2-959b-443a-802d-762757c92373.png)

I can't find where I do this, can anyone point me in the right direction?

Once I get this sorted any tips for copying my current database to the new location in  Home Assistant?

Is there an easier way to just move the entire install to Home Assistant as an add-on?

**Comment by tobiasehlert** [comment]
Hi @shawwellpete,

It totally depends on where you have your PostgreSQL instance running. Is it running together with TeslaMate, then you'd need to check that it's accessible from the Grafana instance, where you want to connect from. And if that is the case, then the host as shown in the picture is faulty, since it should be the connection to that container running on the host on a port that is accessible.

Hope this helps at least a little.. I'm not running HomeAssistant myself, so I can't help you further with that.

Regards,
Tobias

**Reply by shawwellpete** [reply]
Thank you,  The confusing thing is that I have a working Teslamate with PostgreSQL running created from the same stack. I am trying to replicate everything into Home Assistant add-ons where there would be another instance of PostgreSQL as well as Teslamata and Grafana. 

Within Home Assistant I can't see where the option is to configure the database as shown in the image above. I guess I need a Home Assistant user to help for that...
